Document VTTRegion instance properties (#44494)
mdn/content: files/en-us/web/api/vttregion/regionanchorx/index.md

---
title: "VTTRegion: regionAnchorX property"
short-title: regionAnchorX
slug: Web/API/VTTRegion/regionAnchorX
page-type: web-api-instance-property
browser-compat: api.VTTRegion.regionAnchorX
---

{{APIRef("WebVTT")}}

> The **`regionAnchorX`** property of the {{domxref("VTTRegion")}} interface represents the x-coordinate of the region anchor, as a percentage of the region.

Reviewer: ```suggestion
The **`regionAnchorX`** property of the {{domxref("VTTRegion")}} interface represents the x-coordinate of the region anchor, as a percentage of the region's width.
```

I'm assuming this is correct, but it should be specified.

## Value

> A number, in the range `0` to `100`, giving the x-coordinate of the region anchor as a percentage of the region. The default is `0`.

Reviewer: ```suggestion
A number, in the range `0` to `100` inclusive, representing the x-coordinate of the region anchor as a percentage of the region's width. The default is `0`.
```

## Examples

> In the following example a new {{domxref("VTTRegion")}} is created, then the value of `regionAnchorX` is set to `30`. The value is then printed to the console.

Reviewer: ```suggestion
In the following example, a new {{domxref("VTTRegion")}} is created, then the value of `regionAnchorX` is set to `30`. The value is then printed to the console.
```
